README.md

```diff
@@ -218,7 +218,7 @@ app.use('/graphql', graphqlHTTP(request => {
   return {
     schema: MyGraphQLSchema,
     graphiql: true,
-    extensions({ document, variables, operationName, result }) {
+    extensions({ document, variables, operationName, result, context }) {
```

Comment:
Would be great if you put `runTime` in context and make `extension` into a function defined on a top level. It will give realistic usage example for `context` argument